packages/synapse-react-client/src/components/ForumSearch/ForumSearch.tsx

```diff
@@ -150,7 +149,10 @@ export const ForumSearch = (props: ForumSearchProps) => {
             </Typography>
           )}
           {matchList.map(match => (
-            <div key={`${match.forumId}-${match.threadId}-${match.replyId}`}>
+            <div
+              className="bootstrap-4-backport"
```

Comment:
bootstrap-4-backport breaks MUI button styling, but is still required to style the DiscussionReply, so move to its containing div rather than the parent that contains the button

Reply:
Yes! Done in the next commit